tools/topology/sof-cnl-nocodec.m4

```diff
@@ -0,0 +1,69 @@
+#
+# Topology for generic Icelake board with nocodec
+#
+
+# Include topology builder
+include(`utils.m4')
+include(`dai.m4')
+include(`pipeline.m4')
+include(`ssp.m4')
+
+# Include TLV library
+include(`common/tlv.m4')
+
+# Include Token library
+include(`sof/tokens.m4')
+
+# Include Cannonlake DSP configuration
+include(`platform/intel/cnl.m4')
+
+#
+# Define the pipelines
+#
+# PCM0 <---> Volume <---> SSP0 (NoCodec)
```

Comment:
Just wondering why don't we use the same SSP port as sof-cnl-rt5682.m4?

Reply:
@zhuyingjiang @bardliao for nocodec we've usually exposed all SSP ports and not just 1.